### DISCUSSION TITLE
ILS not working, no Magenta text or diamonds

### DISCUSSION THREAD

**Discussion by Riggie** [flybywiresim/aircraft#4315] [Help]
I have the latest FBW A32NX Neo stand-alone experimental installed, and I must thank you for the Fly-pad, it's a godsend for the AirbusThrustmaster quadrant. 
For the last five days, I flew maybe 10, hour-long flights,  and ending with Cat II Cat III ILS landings with no problems at all.
But now the ILS has become problematic, it failed to work once yesterday and today hasn't worked at all,  no Magenta text or diamonds just the 4 white circles horz / vert appear.
Just wondering if anyone else has the same problem, and is there any common reason or error I could be making for it not to appear?
I will try a reinstall later, I think it would be the best way forward.

**Comment by tracernz** [comment]
Have you tuned a localiser on the RAD NAV page?

**Reply by Riggie** [reply]
Hi tracernz, I normally just select the runway that supports ILS on the World Map, just under " Select arrival airport" and that inserts it into the Flight Plan, but I will check that now. Thanks for the help.

**Reply by tracernz** [reply]
> 
> 
> Hi tracernz, I normally just select the runway that supports ILS on the World Map, just under " Select arrival airport" and that inserts it into the Flight Plan, but I will check that now. Thanks for the help.

The auto-tuning works for most airports, but not all. If you run into this issue again check whether there is an auto-filled frequency under LS on the RAD NAV page. You may need to manually enter it at some airports.

**Comment by Riggie** [comment]
Thanks for your help,
As you say the auto-tuning works OK, but not for all airports. Now on the map, I am using the small square icon  ( with the "filter on" ) that appears next to the runway when you click it,  all navigation data is there, so I am just making a note of ILS Freq just in case it doesn't show in Rad Nav. 
thanks again for your help

**Reply by roxter69** [reply]
i have the same problem,even if the ils frequency isnt there and i manually put it in the magenta dosnt always show up,another problem i have had is the decel point is about a inch of my flight path,sometimes it captures and other times it dosnt,this has been happening for a while now,im using the latest experimental version
